inventory/urls.py

```diff
@@ -0,0 +1,39 @@
+from django.urls import path, include
+
+from inventory.views import *
+
+namespace = 'inventory'
+
+urlpatterns = [
+    # Change Language helper view
+    path('', BoardGameView.as_view(), name='home'),
+    path('my_items/', include([
```

Comment:
I agree that moving the URLs to a different URL-system is definitely a no-go. Functionality is provided by `inventory`, so the URLs should be provided by `inventory` (and circular imports would be a problem too, like you mentioned). It would be possible to replace
```path('inventory/', include(('inventory.urls', 'inventory'), namespace='inventory')),```
by
```path('', include(('inventory.urls', 'inventory'), namespace='inventory')),```
in `squire/urls.py` so that not all URLs included in `inventory/urls.py` need to start with `inventory/`. This currently also happens in `membership_file` (with `account/membership` vs. `/no_member` both being provided there) and in `activity_calendar` (with `api/calendar/ical` vs. `calendar/`).